variables.tf

```diff
@@ -27,6 +27,12 @@ variable "sm_service_plan" {
   }
 }
 
+variable "create_iam_engine" {
```

Comment:
The only thing getting "created" here is s2s auth policies between SM and IAM. So maybe we need to rename this to `skip_iam_authorization_policy` ?

The variable description needs to be more specific as to what exact type of auth policies are created (as its possible a consumer may already have them set in their account)